framework/doc/content/source/meshgenerators/FileMeshGenerator.md

```diff
@@ -72,6 +73,8 @@ The conversion capabilities to be able to read those files are summarized here f
 | .xml        | DOLFIN xml  | meshio          | Exodus |
 | .case       | EnSight     | Paraview        | Exodus |
 | .f3grid     | FLAC3D      | meshio          | Exodus |
+| .grdecl     | Eclipse     | em2ex           | Exodus |
+| -           | Leapfrog Geothermal | em2ex   | Exodus |
```

Comment:
`-` ?

Reply:
there s no official extension mentioned in the em2ex doco

Reply:
Should just be CSV (https://github.com/cpgr/em2ex/blob/master/README.md)

Reply:
ok changed. This might be confusing to users (.csv is kind of a lot of things). time will tell